### PR TITLE
Update MainMenu.cs to use ISawmill

### PR DESCRIPTION
**Pull request by Winkarst-cpu**
## About the PR

Title.

## Why / Balance

To remove warnings, ``Logger`` logging methods are obsolete.

## Technical details

Changed ``Logger`` to ``ISawmill``.

## Media

There is nothing to show.

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

None.

### CODE REVIEW

File: Content.Client/MainMenu/MainMenu.cs [hunk 1]
```diff
@@ -26,6 +26,8 @@ public sealed class MainScreen : Robust.Client.State.State
         [Dependency] private readonly IResourceCache _resourceCache = default!;
         [Dependency] private readonly IUserInterfaceManager _userInterfaceManager = default!;
 
+        private readonly ISawmill _sawmill = Logger.GetSawmill("mainmenu");
+
```
Comment (metalgearsloth) [inline Content.Client/MainMenu/MainMenu.cs:30]: Is this already a dependency? I don't see ILogManager anywhere.

Comment (Winkarst-cpu) [reply Content.Client/MainMenu/MainMenu.cs:30]: Uh, it's Logger class.

https://github.com/space-wizards/RobustToolbox/blob/fc1cca4f48f2f2d3fbf41aa42b80b4e43b1095a4/Robust.Shared/Log/Logger.cs#L16

Comment (metalgearsloth) [reply Content.Client/MainMenu/MainMenu.cs:30]: Can't tell on github if it's the static or not.

Comment (Winkarst-cpu) [reply Content.Client/MainMenu/MainMenu.cs:30]: > Can't tell on github if it's the static or not.

https://github.com/space-wizards/RobustToolbox/blob/4920ecaa64310a1680c388dd21c7ee7a43590bc7/Robust.Shared/Log/Logger.cs#L16

**Pull request merged**